$NetBSD: patch-af,v 1.1.1.1 2009/04/20 21:27:40 tron Exp $

Add support of existence of header files:
	machine/byte_swap.h and net/pf/pfvar.h.

--- include/autoconf.h.in.orig	2009-04-11 17:35:37.000000000 +0900
+++ include/autoconf.h.in
@@ -306,6 +306,9 @@
 /* Define to 1 if you have the `lrand48' function. */
 #undef HAVE_LRAND48
 
+/* Define if you have the <machine/byte_swap.h> header file.  */
+#undef HAVE_MACHINE_BYTE_SWAP_H
+
 /* Define to 1 if you have the `mallinfo' function. */
 #undef HAVE_MALLINFO
 
@@ -393,6 +396,9 @@
 /* Define to 1 if you have the <net/pfvar.h> header file. */
 #undef HAVE_NET_PFVAR_H
 
+/* Define to 1 if you have the <net/pf/pfvar.h> header file. */
+#undef HAVE_NET_PF_PFVAR_H
+
 /* off_t is defined by the system headers */
 #undef HAVE_OFF_T
 
